### PR TITLE
create schema

### PR DESCRIPTION
**Pull request by thomascchen**

### CODE REVIEW

File: db/migrate/20150629201312_create_beaches.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class CreateBeaches < ActiveRecord::Migration
+  def change
+    create_table :beaches do |t|
+      t.string :street1, null: false
+      t.string :street2, null: false
+      t.string :city, null: false
+      t.string :state, null: false
+      t.string :zip, null: false
+      t.string :name, null: false
+      t.integer :entrance_fee
```
Comment (dpickett) [inline db/migrate/20150629201312_create_beaches.rb]: will you express entrance fees as the price in cents? If not, you should use use a `double` datatype with appropriate scale and precision

File: db/migrate/20150629201312_create_beaches.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      t.string :picture_url
+      t.text :description
+      t.integer :user_id, null: false
```
Comment (dpickett) [inline db/migrate/20150629201312_create_beaches.rb:13]: foreign keys should be indexed

File: db/migrate/20150629202419_create_beach_amenities.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateBeachAmenities < ActiveRecord::Migration
+  def change
+    create_table :beach_amenities do |t|
+      t.integer :beach_id, null: false
```
Comment (dpickett) [inline db/migrate/20150629202419_create_beach_amenities.rb:4]: foreign keys should be indexed

File: db/migrate/20150629202535_create_reviews.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.integer :rating, null: false
+      t.text :description
+      t.integer :beach_id, null: false
```
Comment (dpickett) [inline db/migrate/20150629202535_create_reviews.rb:6]: can a user review the same beach more than once? If not, we should add a unique index for `[:beach_id, :user_id]` and validate uniqueness

File: db/migrate/20150629202819_create_votes.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.integer :user_id, null: false
```
Comment (dpickett) [inline db/migrate/20150629202819_create_votes.rb:4]: can a user vote on the same review more than once? if not, we should add a unique index and validate uniqueness

File: db/migrate/20150629202941_create_amenities.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
+class CreateAmenities < ActiveRecord::Migration
+  def change
+    create_table :amenities do |t|
```
Comment (dpickett) [inline db/migrate/20150629202941_create_amenities.rb:3]: this does not adhere to the revised, approved ER diagram

**Comment by thomascchen** [comment]
indexes will be added by their implementor tomorrow

**Pull request merged**